pkgs/data/fonts/readexpro/default.nix

```
@@ -0,0 +1,49 @@
+{
+  pkgs ? import <nixpkgs> {},
+  lib ? import <nixpkgs/lib>,
+  fetchgit ? pkgs.fetchgit,
+  stdenvNoCC ? pkgs.stdenvNoCC,
+}:
```

pkgs et al. are already available within the context of a package definition:

```suggestion
{
  pkgs,
  lib,
  fetchgit,
  stdenvNoCC,
}:
```